python/tests/test_table.py

```diff
@@ -206,12 +207,17 @@ def test_polars(db):
     )
     assert table.schema == schema
 
+    # search results to polars dataframe
     q = [3.1, 4.1]
     result = table.search(q).limit(1).to_polars()
     assert np.allclose(result["vector"][0], q)
     assert result["item"][0] == "foo"
     assert np.allclose(result["price"][0], 10.0)
 
+    # enter table to polars dataframe
+    result = table.to_polars()
```

Comment:
The default scan_pyarrow_dataset will pushdown filters.  Do you want to add a test case with a filter to see if that breaks things?

Reply:
hmmm...unfortunately pushdown doesn't work since it pushes down compute expressions.
So if you have an equality filter, the str repr of the compute expr parses so that works, but something `is_in` does not. I guess we need to hard code it to False for now?